.phpcs.xml.dist

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0"?>
+<ruleset
+		name="VIP Go Sites"
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:noNamespaceSchemaLocation="vendor/squizlabs/php_codesniffer/phpcs.xsd">
+	<description>Custom ruleset for VIP Go sites.</description>
+
+	<!-- For help in understanding this file: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml -->
+	<!-- For help in using PHPCS: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Usage -->
+	<!-- For VIP GO code review items: https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/ -->
+	<!-- For VIP GO PHPCS severity levels: https://vip.wordpress.com/documentation/phpcs-review-feedback/ -->
+
+	<!-- What to scan -->
+	<file>./themes/</file>
+	<!-- Ignoring Files and Folders:
+		https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-files-and-folders -->
+	<exclude-pattern>./themes/twentyseventeen/</exclude-pattern>
+	<exclude-pattern>./node_modules/</exclude-pattern>
+	<exclude-pattern>./vendor/</exclude-pattern>
+
+	<!-- How to scan -->
+	<arg value="sp"/> <!-- Show sniff and progress -->
+	<arg name="colors"/> <!-- Show results with colors. Disable if working on Windows -->
```

Comment:
Might be better to start with this one commented out, and let developers enable it themselves (or use an alias such that `phpcs = "phpcs --colors"`.